content/en/cloud_cost_management/google_cloud.md

```diff
@@ -0,0 +1,124 @@
+
+{{< beta-callout url="https://docs.google.com/forms/d/e/1FAIpQLScNSzkTAmmpmxEY5Xoo-xPOyKXG0n_6IgdlU66mWXp445pNdA/viewform" btn_hidden="false" >}}
+Cloud Cost Google Cloud support is in private beta
+{{< /beta-callout >}}
+
+## Overview
+
+To use Cloud Cost Management with Google Cloud, you must set up a Datadog Google Cloud Platform Integration in Datadog. You must also enable a Detailed Usage Cost billing export for your desired billing account in Google Cloud.
+
+The Google Cloud Platform Datadog Integration Service Account must have necessary permissions configured to interact with Google Cloud Storage and BigQuery.
+
+## Setup
+
+### Enable Detailed Usage cost billing export
+<div class="alert alert-info"> <a href="https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage">Detailed Usage Cost</a> data includes resource-level cost data. </div>
+
+1. Navigate to [Billing Export][1] under Google Cloud console *Billing*
+2. Enable [*Detailed Usage cost*][2] export, selecting or creating a project and a BigQuery dataset
+3. Document the billing account id for the billing account where the export was configured as well as export project id and dataset name
+
+### Create or select a Google Cloud Storage bucket
+You can use an existing GCS bucket or create a new one. Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to a `datadog_cloud_cost_detailed_usage_export` prefix in the selected bucket.
+
+**Note:** The bucket [must be co-located][9] with the BigQuery export dataset.
+
+### Configure the Google Cloud Platform integration
+Navigate to [Setup & Configuration][3] and select an integrated Google Cloud Platform service account from the menu. If you do not see your desired Service Account in the list, go to [Google Cloud Platform integration][4] to add it. Follow the instructions on the integration page to properly configure your Service Account being integrated.
+
+<div class="alert alert-warning"> LEGACY project integrations are deprecated and not supported </div>
+
+Datadog Google Cloud Platform Integrations monitor the entire project when a related service account is integrated. Using a previously integrated project prevents monitoring resources in a new project. If your billing is associated with a non-integrated project, those resources will start to be monitored.
+
+**Note:** You can limit Metrics Collection per integration - this limits automatically monitoring resources in the project - but does not exclude data from Cloud Cost processing. </div>
+
+### Provide Datadog integration service account necessary permissions
+The APIs and permissions below enable Datadog to access your Detailed Usage billing export data and extract it in a useful format. This dumps data from BigQuery, where Google Cloud exports it, to your specified GCS bucket through a scheduled BigQuery query. The data is exported in a useful manner in your cloud storage bucket where it can then be processed by Datadog.
```

Comment:
```suggestion
The APIs and permissions below enable Datadog to access your Detailed Usage billing export data and extract it in a useful format. This exports data from BigQuery, where Google Cloud then exports it to your specified GCS bucket through a scheduled BigQuery query. The data is exported in a useful manner in your cloud storage bucket where it can then be processed by Datadog.
```

Reply:
So when a user sets up the detailed usage export documented here, GCP exports that billing data to BigQuery. This process helps us get the data out of BigQuery and into a format we can process it regularly

Reply:
Got it! I'll mark it as resolved :)